test/packaging/ansible/roles/initial-install/tasks/repo-Amazon.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
should we run an analog of `apt-get update` to check that repo is fine here?

Reply:
the only thing this does is adding the repo, so it'll complain and fail in case it doesn't exist when installing the package

Reply:
👍